src/pages/api/logs/[id].js

```diff
@@ -4,15 +4,21 @@ import {
   deleteLog,
   getLogById,
 } from "../../../../server/db/actions/Log";
+import { getUserById } from "../../../../server/db/actions/User";
 import { z } from "zod";
 import { consts } from "@/utils/consts";
+import { getServerSession } from "next-auth/next";
+import { authOptions } from "../auth/[...nextauth].js";
 
 const logSchema = z.object({
```

Comment:
we have this in the `consts.js`

Reply:
This logSchema excludes author and dog, which was causing problems for me during my PATCH request, is it ok if i keep it?